src/components/abstractComponent.ts

```diff
@@ -95,7 +95,10 @@ export module Component {
     }
 
     public _requestedSpace(availableWidth : number, availableHeight: number): _SpaceRequest {
-      return {width: 0, height: 0, wantsWidth: false, wantsHeight: false};
+      return {
+        width: 0,
+        height: 0
+      };
```

Comment:
The "sometimes one-liner" seems less enforceable. While putting every property on a different line is less compact, I think it makes it clearer what the properties are (and by extension, makes it easier to identify the type of object being returned).